h/api/models/elastic.py

```diff
@@ -58,7 +64,28 @@ def target_links(self):
     def document(self):
         return self.get("document", {})
 
+    def __acl__(self):
```

Comment:
Is this something we need to add to the Postgres model as well? (Either now or soon)